cloud-function/README.md

```diff
@@ -47,9 +47,8 @@ The placement handler uses the following environment variables:
 - `KEVEL_NETWORK_ID` (default: `0`) - Required for serving placements via Kevel.
 - `SIGN_SECRET` (default: `""`) - Required for serving placements.
 - `BSA_ZONE_KEYS` (default: `""`) - Required for serving placements via BSA.
-- `BSA_URL_PREFIX`(default: "https://localhost") - Where to show BSA placements
-  if enabled. Formatted like :
-  "placementname1:zonekey1;placementkey2:zonekey2...".
+- `BSA_URL_PREFIX`(default: `""`) - Where to show BSA placements if enabled.
+  Formatted like : "placementname1:zonekey1;placementkey2:zonekey2...".
```

Comment:
Not sure how, but these lines got mixed up in a prior commit:
```suggestion
  Formatted like : "placementname1:zonekey1;placementkey2:zonekey2...".
- `BSA_URL_PREFIX`(default: `""`) - Where to show BSA placements if enabled.
```

Reply:
I wonder also if the `BSA_URL_PREFIX` needs some additional explanation, like that it needs to include proto+hostname?

Reply:
I have now removed that environment variable in 525284ae8.